Makefile

```diff
@@ -1,8 +1,20 @@
-setup-libs:
+.PHONY: npm-install
+npm-install:
 	npm install
 
-site:
+.PHONY: npm-start
```

Comment:
@Aisuko we use `make site` in all 6 of the websites managed by the community.

Reply:
I will replace the `npm-start`

Reply:
I did a new PR to the new master branch, Should we work on that one or go on this one?